build/blocks/date-block/block.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "gatherpress/date-block",
+  "version": "0.1.1",
+  "title": "GP Date Block",
+  "category": "gatherpress",
+  "icon": {
+    "background": "#29c8aa",
+    "foreground": "white",
+    "src": "flag"
+  },
+  "description": "A block to build from! This block enables you to type text and style it with the color font Gilbert from Type with Pride.",
+  "attributes": {
+    "theTime": {
+      "type": "string"
+    },
+    "message": {
+      "type": "string",
+      "source": "text",
+      "selector": "div"
+    }
+  },
+  "supports": {
+    "html": false
+  },
+  "textdomain": "gatherpress",
+  "editorScript": "file:./index.js",
+  "editorStyle": "file:./index.css",
+  "style": "file:./style-index.css"
+}
```

Comment:
convert from spaces to tabs (tabs is WordPress standard)

Reply:
Never mind the space/tabs comment, just realized this is the build file not the src... oops 😂